config.json

```diff
@@ -15,6 +15,9 @@
     "indent_size": 4,
     "highlightjs_language": "armasm"
   },
+  "test_runner": {
+    "average_run_time": 3
```

Comment:
Other tracks set this to `2`, but probably only because this is what's in the [example](https://exercism.org/docs/building/tracks/config-json#h-example). We could probably measure this to get a realistic number. I don't know what this is used for anyway, so going with 3 for now is probably fine.